io.github.pieterdd.StretchBreak.yaml

```diff
@@ -0,0 +1,43 @@
+id: io.github.pieterdd.StretchBreak
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: stretch-break
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--device=dri"
+- "--env=G_MESSAGES_DEBUG=none"
+- "--env=RUST_BACKTRACE=1"
+- "--talk-name=org.gnome.Mutter.IdleMonitor"
+- "--talk-name=org.freedesktop.ScreenSaver"
+- "--talk-name=org.kde.ScreenSaver"
+- "--talk-name=org.freedesktop.Notifications"
+modules:
+  - name: stretch-break
+    buildsystem: simple
+    build-options:
+      append-path: "/usr/lib/sdk/rust-stable/bin"
+      env:
+        CARGO_HOME: "/run/build/stretch-break/cargo"
+    build-commands:
+    - cargo --offline fetch --manifest-path Cargo.toml --verbose
+    - cargo --offline build --release --verbose
+    - install -Dm755 ./target/release/stretch-break /app/bin/stretch-break
+    - install -Dm644 meta/logo-color-16x16.png /app/share/icons/hicolor/16x16/apps/io.github.pieterdd.StretchBreak.png
+    - install -Dm644 meta/logo-color-32x32.png /app/share/icons/hicolor/32x32/apps/io.github.pieterdd.StretchBreak.png
+    - install -Dm644 meta/logo-color-128x128.png /app/share/icons/hicolor/128x128/apps/io.github.pieterdd.StretchBreak.png
+    - install -Dm644 -t /app/share/applications/ meta/io.github.pieterdd.StretchBreak.desktop
+    - install -Dm644 -t /app/share/metainfo/ meta/io.github.pieterdd.StretchBreak.metainfo.xml
+    - install -Dm644 -t /app/share/dbus-1/services/ meta/io.github.pieterdd.StretchBreak.Core.service
+    - sed -i 's|/usr/bin/stretch-break|/app/bin/stretch-break|g' /app/share/dbus-1/services/io.github.pieterdd.StretchBreak.Core.service
```

Comment:
thing is this should be dealt with the upstream build system and even `/usr` is never the default prefix. This would include installing all the above. There are many GNOME application in Rust that build with meson.

Reply:
Meson is not a good fit for me, it's way too complex for what I need. If you really don't want this in the Flatpak build, I'll make a separate file with the Flatpak path in the upstream repo.